app/controllers/contact_infos_controller.rb

```diff
@@ -41,7 +41,7 @@ def resend_confirmation
                   path = current_user.is_email_pending? ? verification_sent_path :
                                                           profile_path(active_tab: :email)
```

Comment:
Oh... it's probably always true.  I think maybe I'm looking for `current_user.is_temp?`.